hw/ip/aes/doc/_index.md

```diff
@@ -328,6 +328,14 @@ Future versions of this AES unit thus might employ different means at architectu
 This section discusses how software can interface with the AES unit.
 
 
+## Clear upon Reset
+
+Upon reset, the AES unit will first reseed the internal PRNG for register clearing via EDN, and then clear all key, IV and data registers with pseudo-random data.
```

Comment:
"... and then fill ... " or "initialize" ?

Reply:
I very much favor consistency between documentation and RTL. Please refer to the [other discussion](https://github.com/lowRISC/opentitan/pull/5651/files#r595892421) for more detail.